client/app/components/user/session.service.js

```diff
@@ -438,14 +448,20 @@ class SessionService {
                 ];
             }
 
+            if (notificationsMenu.show) {
+                baseNavbar.internalLinks.splice(1, 0, notificationsMenu);
+            }
+
             return baseNavbar;
         };
 
-        return this.$q.all({
-            translate: this.loadTranslations(),
-            user: this.user.getUser()
-        })
-            .then(({ user }) => getBaseNavbar(user))
+        return this.$q
+            .all({
+                translate: this.loadTranslations(),
+                user: this.user.getUser(),
+                notifications: this.navbarNotificationService.getNavbarContent()
+            })
+            .then(({ user, notifications }) => getBaseNavbar(user, notifications))
```

Comment:
Whats `translate` for then ? You need it to be resolved before `getBaseNavbar` but you don't need the result, right ?

Reply:
probably, but it avoid problems because of all those single binding in the different managers and I want my code to be the same between all managers